netcontrol/arp.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+class Arp:
+    """
+    Class which interacts with the ARP table
+    """
+    def __init__(self, logger: logging.Logger):
+        self.logger = logger
+
+    def get_mac(self, ip: str):
+        """
+        Get the mac address associated with a given ip address.
+
+        :param ip: Ip address of the machine.
+        :return: Mac address of the machine.
+        """
+        self.logger.info("Querying MAC for IP %s", ip)
+        f = open('/proc/net/arp', 'r') # Open arp table
```

Comment:
You never close the file, it's better to user `with open(...) as f:` to prevent resources waste.